pkg/server/server.go

```diff
@@ -28,7 +26,6 @@ func Setup(r *gin.Engine, c *Config) {
 	r.Use(middleware.SetSQLClient(c.SQLClient))
 	r.Use(middleware.SetKubeController(c.KubeController))
 	r.Use(middleware.SetArtifactCredentialsController(c.ArtifactCredentialsController))
-	r.Use(middleware.SetKubeActionHandler(c.KubeActionHandler))
```

Comment:
If I understand, now the kube handler is no longer used as a middleware, now, we can directly call those actions with the new implementation?

Reply:
That's correct.